components/camel-splunk-hec/src/main/java/org/apache/camel/component/splunkhec/SplunkHECConfiguration.java

```diff
@@ -36,6 +36,8 @@ public class SplunkHECConfiguration {
     private String source = "camel";
     @UriParam
     private String host;
+    @UriParam(defaultValue = "/services/collector/event")
+    private String splunkRESTEndpoint = "/services/collector/event";
```

Comment:
Cane we use `splunkEndpoint` instead as upper-case REST make it a bit more unusual and a bit harder to read.

Reply:
Yes, I'll change it.

Reply:
Thanks that was quick